darwin/DarwinProcess.c

```diff
@@ -9,13 +9,63 @@ in the source distribution for its full text.
 
 #include "darwin/DarwinProcess.h"
 
+#if !TARGET_OS_IPHONE
```

Comment:
As @Explorer09 mentioned: This should be detected via `configure`.

Also with headers that need defines for their inclusion, the inclusion order changes slightly (cf. style guide).

Reply:
No longer applicable.